float64 UTM_lat
float64 UTM_lon
int16 bearing

